### ISSUE TITLE
String formatting with eV?

### ISSUE THREAD

**Issue by SimonHeybrock** [LLNL/units#121]
#112 fixed some string formatting for units containing `meV` (milli electron volt). In our field (time-of-flight neutron scattering) we also encounter some odd combinations with this, the main example comes to mind is `counts/meV`. Currently this gets formatted badly:

```cpp
// parsing ok
unit_from_string("counts/meV")
// to_string not ok
6.2415090744607627e+21counts/J
```

Is there a generic way to fix this, that doesn't require adding more exceptions?

**Comment by phlptp** [comment]
I will take a look, I am looking at some issues like this

**Comment by SimonHeybrock** [comment]
Fixed by #130 (thanks!), but I have a related question: In fields where `eV` is used for energy, there is a set of related units than contain additional odd factors, in particular `c` (speed of light). See for example table on top right here: https://en.wikipedia.org/wiki/Electronvolt.

The most relevant ones are `eV/c**2` for mass and `eV/c` for momentum.

Would you consider adding support for this? If so, I could give it a try myself (maybe you have some pointers on what is the best way?).

**Comment by phlptp** [comment]
My guess is adding `c`, `c^2` and `eV` and possibly `h` and `hbar` to the `test_units` structure on line 319 of `units.cpp` might deal with a number of cases like that pretty easily.
